docs/index.rst

```diff
@@ -40,11 +40,11 @@ contacting support.
 +--------------------------------------------------+---------------+----------------+
 |                   Integration                    |  |SUPPVER|    | |AUTO|    [1]_ |
 +==================================================+===============+================+
-| :ref:`aiobotocore`                               | >= 0.2.3      | No             |
```

Comment:
we also need to update the corp docs: https://github.com/DataDog/documentation

we should also consider adding the note about library version support to the corp docs as well.

Reply:
As per offline discussion: we are only dropping testing for older versions, not official support. We won't need to update public docs yet, at least until 2.0.